Added a note to clear potential confusion. (#253)
mulesoft/docs-access-management: modules/ROOT/pages/conf-openid-connect-task.adoc

+
. Log in through your identity provider to test the configuration.

> Note, the dynamically registered application at the identity provider has only default settings. If you want to configure additional functionalities e.g. groups mapping, you will be responsible to update the settings on the provider side. Configuring dynamically registered applications is not currently supported. 

Reviewer: ```suggestion
[NOTE]
The dynamically registered application at the identity provider has only default settings. If you want to configure additional functionalities (such as group mappings), you must update the settings on the provider side. Configuring dynamically registered applications is not supported. 
```
